src/Mod/Path/LibLathe/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
What is this LibLathe exactly? Is this an external library?

FreeCAD is licensed LGPL2, therefore, all files that are included in the source code must be licensed LGPL2 as well, or be compatible with it (MIT, BSD). This LibLathe library cannot be included in the source code if it has GPL2 license. This library can be installed and used at run time, but it cannot be provided together with FreeCAD. For example, see the DXF library that is used in the Draft Workbench. This library can be installed from an online repository, but it is not included together with the Draft Workbench.

Alternatively, if the LibLathe is re-licensed to LGPL2 by its author, then it can be included in FreeCAD without problems.